README.md

```diff
@@ -11,6 +11,7 @@ It supports Python 2.7 and 3.5+.
 You should consider updating your Python version to a supported one.
 
 [![Tests Status](https://github.com/python-poetry/poetry/workflows/Tests/badge.svg?branch=master&event=push)](https://github.com/python-poetry/poetry/actions?query=workflow%3ATests+branch%3Amaster+event%3Apush)
+[![Version](https://img.shields.io/pypi/v/poetry.svg)](https://img.shields.io/pypi/v/poetry.svg)
```

Comment:
Link should point to pypi release. Is it possible to have two badges? One for stable and another for pre-release?